### ISSUE TITLE
Improve user interface of site settings page

### ISSUE THREAD

**Issue by crookedneighbor** [Commander-Spellbook/website-v2#227]
Blocked by https://github.com/Commander-Spellbook/website-v2/pull/213

Acceptance criteria:

- [ ] set page in a loading state on initial load before the site settings data has loaded from firebase
- [ ] set pate in a loading state while the api call to update the feature combos is being made

**Comment by crookedneighbor** [comment]
No longer relevant as the backend system is happening in a separate app now.
